packages/lib/gpcircuits/test/proto-pod-gpc.spec.ts

```diff
@@ -400,12 +439,69 @@ function makeTestSignals(
     sigEntryProofDepth.push(BigInt(entrySignals.proof.siblings.length));
     sigEntryProofIndex.push(BigInt(entrySignals.proof.index));
 
-    // Fillin sibling array, padded with 0s to max length.
+    // Fill in sibling array, padded with 0s to max length.
     sigEntryProofSiblings.push(
       extendedSignalArray(entrySignals.proof.siblings, params.merkleMaxDepth)
     );
   }
 
+  // A list of pairs of indices and values.
+  // The values will be zipped together to form the
+  // actual membership list.
+  const listData = [
+    [
+      0,
+      [sigEntryValue[0], 85n, 0n].map((x) => {
+        return { type: "cryptographic", value: x };
+      })
+    ],
+    [
+      3,
+      [sigEntryValue[3], 876n, 999n].map((x) => {
+        return { type: "int", value: x };
+      })
+    ],
+    [
+      4,
+      [sigEntryValue[4], 103n, 1n].map((x) => {
+        return { type: "int", value: x };
+      })
+    ]
+  ]
+    .map((pair) => pair as [number, PODValue[]])
+    // Consider only those entry values that are enabled.
+    .filter((pair) => sigEntryIsValueEnabled[pair[0]] === 1n)
```

Comment:
I think this is unnecessary.  sigEntryIsValueEnabled indicates that the plaintext value is in the circuit, which only works for numbers.  Tuples and memberrship checks should work for strings and other values too.